app/views/eligibility-tool/index.html

```diff
@@ -9,14 +9,23 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-xl">
-        Check if you can get shared parental leave and pay
+        Check if you can get Shared Parental Leave and Pay
       </h1>
       <p>
-        Use this tool to check if you or your partner can get <a href="/shared-parental-leave-and-pay">shared parental leave and pay</a> in the first year of your child's life.
+        Use this tool to check if you or your partner can get <a href="/shared-parental-leave-and-pay">Shared Parental Leave and Pay</a>.
       </p>
       <p>
-        You can check eligibility for each parent individually, but in either case you'll need to answer questions on work and earnings for <strong>both</strong> parents.
+        You can check eligibility for each parent individually, but in either case you'll need to answer questions on work and earnings for both parents.
       </p>
+      <p>
+        Before you start, you'll need to know:
+      </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Your child’s due date, birth date or placement date</li>
+      <li>Employment status for the parent whose eligibility you’re checking </li>
```

Comment:
Remove the traling space before the close tag:
`checking </li>`
to
`checking</li>`